Description: Remove deprecated PLASTIC support
 PLatform for AStronomical InterConnection (PLASTIC)
 was deprecated as a local IPC mechanism, and superseded by the
 Simple Applications Messaging Protocol (SAMP).
 By disabling Plastic support dependencies are removed.
 This patch disables compliation, disables internal support,
 and patches out remaining references within those these
 disabled/dead-codepaths.
Author: Paul Sladen <debian@paul.sladen.org>
Last-Updated: 2016-07-07
Index: aladin/build.xml
===================================================================
--- aladin.orig/build.xml	2016-12-19 11:59:35.553736450 +0100
+++ aladin/build.xml	2016-12-19 11:59:36.233736436 +0100
@@ -16,6 +16,7 @@
      <mkdir dir="${bin}"/>
      <javac srcdir="${basedir}/cds" destdir="${bin}" debug="true"
 	    encoding="Cp1252" deprecation="off" includeantruntime="true">
+       <exclude name="**/PlasticManager.java" />
        <classpath>
           <pathelement path="${java.class.path}/"/>
           <pathelement path="jsamp.jar"/>
Index: aladin/cds/aladin/Aladin.java
===================================================================
--- aladin.orig/cds/aladin/Aladin.java	2016-12-19 11:51:39.493746219 +0100
+++ aladin/cds/aladin/Aladin.java	2016-12-19 11:59:36.237736436 +0100
@@ -5776,21 +5776,12 @@
          if( USE_SAMP_REQUESTED ) {
             appMessagingMgr = new SAMPManager(this);
          }
-         else if( USE_PLASTIC_REQUESTED ) {
-            appMessagingMgr = new PlasticManager(this);
-         }
          // TODO : test if hub is responding !!
          // else look for an existing conf file
          else if( SAMPManager.getLockFile().exists() ) {
             appMessagingMgr = new SAMPManager(this);
          }
-         else if( PlasticManager.getLockFile().exists() ) {
-            appMessagingMgr = new PlasticManager(this);
-         }
          // else take default
-         else if( DEFAULT_MESSAGING_MGR.equals(PlasticManager.class) ) {
-            appMessagingMgr = new PlasticManager(this);
-         }
          else {
             appMessagingMgr = new SAMPManager(this);
          }
